doc/source/cookbook.rst

```diff
@@ -1333,20 +1314,20 @@ The :ref:`Timedeltas <timedeltas.timedeltas>` docs.
 
    s.max() - s
 
-   s - datetime.datetime(2011, 1, 1, 3, 5)
```

Comment:
can you revert those, the import should be `import datetime`

Reply:
Done